gcp_resource_count.sh

```diff
@@ -122,6 +126,23 @@ for project in $PROJECTS; do
       echo "Container Hosts Count: $project_nodes_count"
     fi
 
+    # Fetch Cloud SQL databases
+    gcloud -q sql instances list --project "${project}" --format json > ${_temp_project_output} 2>> $LOG_FILE || echo "Failed to get Cloud SQL DBs for project ${project}"
```

Comment:
how about the type we skip using FF at the moment?

Reply:
Can't really support it since we change it with FF, this doesn't need to be 100% accurate, it's an estimation, we don't have many DBs skipped this way

Reply:
Also we should be able to support them if we take this ticket
https://orcasecurity.atlassian.net/browse/DSEC-1561
so we should count them in calculations anyway